samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/AssessmentSettingsBean.java

```diff
@@ -1276,6 +1287,41 @@ public void setFeedbackDateString(String feedbackDateString) {
     }
   }
 
+  public String getFeedbackEndDateString() {
+    if (!this.isValidFeedbackEndDate) {
+      return this.originalFeedbackEndDateString;
+    } else {
+      return getDisplayFormatFromDate(feedbackEndDate);
+    }
+  }
+
+  public void setFeedbackEndDateString(String feedbackEndDateString) {
+    if (StringUtils.isEmpty(feedbackEndDateString)) {
```

Comment:
I noticed there's a lot of uses of `StringUtils.isEmpty()` and `StringUtils.isNotEmpty()`, however I think it would be more appropriate to use `isBlank()` and `isNotBlank()` so that a string like `" "` is also considered to be "empty".